website/docs/error-reference.md

```diff
@@ -1126,6 +1126,23 @@ def publish_item(name, category = nil)
 end
 ```
 
+## 7013
+
+Sorbet does not support constant reassignment:
```

Comment:
It actually does, it just requires that they be the same type. Do you think that's worth mentioning?

Reply:
#### Input

[→ View on sorbet.run](https://sorbet.run/#%23%20typed%3A%20strict%0AFoo%20%3D%201%0AFoo%20%3D%202)

```ruby
# typed: strict
Foo = 1
Foo = 2
```

#### Observed output

```
No errors! Great job.
```